docs/UserGuide.md

```diff
@@ -85,41 +87,50 @@ Format: `add id/STUDENTID n/NAME p/PHONE c/COURSE yr/SENIORITY avail/AVAILABILIT
 Examples:
 * `add id/E0123456 n/John Doe p/87654321 c/Computer Science yr/2 avail/1,2,3` adds a new TA with Student ID, E0123456, named John Doe.
 
-### Listing all Teaching Assistants : `list`
+### Editing a Teaching Assistant: `edit`
 
-Displays all TAs found in the system.
+Edits a TA in the system.
 
-Format: `list`
+Format: `edit INDEX c/COURSE yr/YEAR [ATTRIBUTE_FIELD/VALUE]...`
 
-<!-- For Edit TA in the future
-### Editing a candidate : `edit`
+<div markdown="block" class="alert alert-info">
 
-Edits an existing candidate in the address book.
+**:information_source: Notes about the edit format:**<br>
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+`ATTRIBUTE_FIELD` can take on the following values `id`, `name`, `phone`, `course`, `yr`, `as`, `avail`
+
+* `as` is short for `ApplicationStatus`.
+
+</div>
 
-* Edits the candidate at the specified `INDEX`. The index refers to the index number shown in the displayed candidate list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the candidate at the specified `INDEX`. The index refers to the index number shown in the displayed TA list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the candidate will be removed i.e. adding of tags is not cumulative.
-* You can remove all the candidate’s tags by typing `t/` without
-    specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st candidate to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd candidate to be `Betsy Crower` and clears all existing tags.
-*  `edit 3 as/Accepted is/Interviewing` Edits the Application Status and Interview Status of the Candidate.
--->
+* `edit 1 n/Jane Doe yr/3 avail/1` Edits the name, year and availability of the 1st candidate to be Jane Doe, Year 3, Monday only respectively.
+* `edit 2 c/Business Analytics` Edits the course of the 2nd candidate to be `Business Analytics`.
+
+### Listing all Teaching Assistants : `list`
```

Comment:
Same for here too!

Reply:
Alright @tiewweijian! Will update all to candidates, which means we gotta check everything... But I'll fix it up for UG first.